cdf_fabric_replicator/extractor.py

```diff
@@ -65,93 +108,96 @@ def download_files_from_abfss(self, abfss_directory: str) -> None:
                 if state and state[0] == file.last_modified.timestamp():
```

Comment:
To make it easier to understand the flow, you can avoid using "continue" like this:

if ( not state || state[0] != file.last_modified.timestamp()):
    # do the upload